homeassistant/components/google/__init__.py

```diff
@@ -191,7 +196,14 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if session.token["expires_at"] >= datetime(2070, 1, 1).timestamp():
         session.token["expires_in"] = 0
         session.token["expires_at"] = datetime.now().timestamp()
+    try:
         await session.async_ensure_token_valid()
+    except aiohttp.client_exceptions.ClientResponseError as err:
```

Comment:
https://docs.aiohttp.org/en/stable/client_reference.html#client-exceptions
```suggestion
    except aiohttp.ClientResponseError as err:
```